internal/resticinstaller/resticinstaller.go

```diff
@@ -242,10 +242,7 @@ func FindOrInstallResticBinary() (string, error) {
 	// Check for restic installation in data directory.
 	resticInstallPath := path.Join(config.DataDir(), resticBinName)
 	if runtime.GOOS == "windows" {
-		programFiles := os.Getenv("programfiles(x86)")
-		if programFiles == "" {
-			programFiles = os.Getenv("programfiles")
-		}
+		programFiles := os.Getenv("programfiles")
 		resticInstallPath = path.Join(programFiles, "restic", resticBinName)
```

Comment:
let's also update this to `backrest` in your PR, I like the suggestion to use the backrest directory for the restic dependency as well.